### DISCUSSION TITLE
Animated back to top button with CSS

### DISCUSSION THREAD

**Discussion by LITUATUI** [thegeeklab/hugo-geekdoc#578] [Ideas]
The idea is to show a back to top button only after scrolling with CSS.

[This](https://www.freecodecamp.org/news/css-only-back-to-top-button/
) article shows some methods that only use CSS, no JS.

I especially like [this](https://codepen.io/t_afif/pen/ExwbZKQ/86f79c96520a0c864324db70a6b76c76) method with a fading button.

What do you guys think?

**Comment by xoxys** [comment]
Can look into it again. On my last try, this wasn't working properly on all Browsers/Window sizes. That's why we decided to implement the current, more simple approach.

**Reply by xoxys** [reply]
The main issue is described here https://moderncss.dev/pure-css-smooth-scroll-back-to-top/#known-issues

> If you have short-form content, this doesn't fail very gracefully on a shorter device viewport.
